libevent-sys/build.rs

```diff
@@ -24,6 +24,11 @@ fn build_libevent(libevent_path: &str) -> PathBuf {
             .define("CMAKE_POLICY_DEFAULT_CMP0066", "NEW")*/
     }
 
+    // TODO: Can we tap into "-vv" cargo argument?
```

Comment:
I tried playing around with the following `build.rs` to see if anything was passed in when doing `-vv`, but I didn't find anything.

```rust
use std::env;

pub fn main() {
    for value in env::args() {
        println!("cargo:warning={}", value);
    }
    for (name, value) in env::vars() {
        println!("cargo:warning={}={}", name, value);
    }
}
```

Reply:
I also drilled into cargo's CLI code for a bit and saw that the flag eventually reaches the [`Shell`](https://docs.rs/cargo/0.44.1/cargo/core/shell/struct.Shell.html) struct which handles the printing, but don't think there's any way to access that.